src/index.ts

```diff
@@ -168,21 +167,26 @@ export class TypeScriptPlugin {
   }
 
   async generateGraphqlTypes() {
-    this.serverless.cli.log('Generate graphql types...');
     const { generates } = this.getGqlCodegenConfig(
       this.originalServicePath,
       this.isWatching ? null : this.serverless.cli
     );
 
-    await gqlCli.generate(
-      {
-        schema: this.graphqlFilePaths,
-        generates,
-      },
-      true
-    );
+    if (generates) {
+      this.serverless.cli.log('Generate graphql types...');
 
-    this.serverless.cli.log('GraphQL Types Generation Complete...');
+      const gqlCli = require('@graphql-codegen/cli');
```

Comment:
won't this conditional require mess up with the bundling jetpack does? Or we'll need to add it as an exception, maybe worth testing it.

Reply:
I tested packaging locally and it worked fine. This is a dev dep, so it's only used during packaging, but it doesn't get included.